rmc-docs/src/limitations.md

```diff
@@ -0,0 +1,413 @@
+| | Operator Traits | Partial | |
+| | `Deref` and `DerefMut` | Yes | |
+| | `Drop` | Partial | |
+| | `Copy` | Yes | |
+| | `Clone` | Yes | |
+14 | Linkage | Yes | |
+15.1 | Unsafe functions | Yes | |
+15.2 | Unsafe blocks | Yes | |
+15.3 | Behavior considered undefined | Partial | |
+| | Data races | No | See [Observations - Concurrency](#concurrency) |
+| | Dereferencing dangling raw pointers | Yes | |
+| | Dereferencing unaligned raw pointers | No | |
+| | Breaking pointer aliasing rules | No | |
+| | Mutating immutable data | No | |
+| | Invoking undefined behavior via compiler intrinsics | Partial | See [Observations - Intrinsics](#intrinsics) |
+| | Executing code compiled with platform features that the current platform does not support | No | |
+| | Producing an invalid value, even in private fields and locals | No | |
+
+## Observations on partially or unsupported features
+
+### Code generation for unsupported features
+
+RMC aims to be a industrial verification tool. Most industrial crates may
+include unsupported features in parts of their code that do not need to be
+verified. In general, this should not prevent users from verifying their code.
+
+Because of that, the general rule is that RMC generates an `assert(false)`
+statement when compiling any unsupported feature. This will cause proofs to fail
+if the statement is reachable during the verification stage. However, the
+analysis will not affected if the statement is not reachable from the code under
+verification, so users can still verify components of their code not using
+unsupported features.
+
+### Assembly
+
+RMC does not support assembly code for now. We may add it in the future but at
+present there are no plans to do so.
+
+Check out the tracking issues for [inline assembly (`asm!`
+macro)](https://github.com/model-checking/rmc/issues/2) and [global assembly
+(`asm_global!` macro)](https://github.com/model-checking/rmc/issues/316) to know
+more about the current status.
+
+### Concurrency
+
+Concurrent features are out of scope for RMC. In general, the verification of
+concurrent programs continues to be an open research problem where most tools
+that analyze concurrent code lack support for other features. Because of this,
+RMC emits a warning whenever it encounters concurrent code and compiles as if it
+was sequential code.
+
+### Standard library functions
+
+At present, RMC is able to link in functions from the standard library but the
+generated code will not contain them. This results in verification failures if
+the code under verification, for example, includes a `println!` statement.
+
+We have done some experiments to embed the standard library into the generated
+code, but this causes verification times to increase significantly. As of now,
+we have not been able to find a simple solution for [this
+issue](https://github.com/model-checking/rmc/issues/581), but we have some ideas
+for future work in this direction.
+
+### Advanced features
```

Comment:
I worry this section might be too vague. It might be a good idea to back-reference what we say in the table? (E.g. there's a lot of links from the table to here that seem to be about projections, and I think our goal is to have to solid by 1.0.)

I think over-technical would be better than vague. If this section answered for us "what would it take for us to remove this section from this doc because we know we support it now" I think that's be better than being vague.

Reply:
I was not sure how to make this more technical, so I highlighted some outstanding issues here.

Please let me know how it looks and if it does not look good to you we can work together on this section.